cogs/endpoints/endpoints.py

```diff
@@ -0,0 +1,7 @@
+"""
+Endpoint data.
+Don't modify this unless you are testing.
+"""
+nekos = {
+    "sfw": "https://nekos.life/api/neko",
+}
```

Comment:
What is this?

Reply:
this is for getting nekos? using the api